shared/util/quit-helper.desktop.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import electron from 'electron'
+import {ipcRenderer, ipcMain} from 'electron'
+import {quit} from '../../desktop/app/ctl'
+
+const app = electron.app || electron.remote.app
```

Comment:
Can we make it so this only works from main process (using ipc) and so we don't have to do this weird .app or remote.app?